onnxruntime/contrib_ops/cpu/cpu_contrib_kernels.cc

```diff
@@ -75,8 +75,8 @@ class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1,
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, uint8_t, QLinearConv);
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, int8_t, QLinearConv);
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, int8_t, MatMulIntegerToFloat);
-class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, int8_t, NhwcMaxPool);
-class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, uint8_t, NhwcMaxPool);
```

Comment:
I guess NhwcMaxPool is generated on-the-fly? I removed these two lines because I wanted to move them to a new domain which is just for NHWC.  I want to copy all layout sensitive ops in ONNX domain to NHWC domain and keep their versions. Currently you don't know which ONNX opset version this NhwcMaxPool op based on.  If we have a model converter that can auto-upgrade all MaxPool op history versions to the latest, then it might be easier to only have one NhwcMaxPool here. And even that it will not have any backward-compatibility capability. When ONNX MaxPool changes,  how we will do here? So even if my this change broke some existing ORT format models, it is an one-time pain that will definitely happen.